src/software/simulated_tests/simulated_tactic_test_fixture.h

```diff
@@ -23,38 +24,75 @@ class SimulatedTacticTestFixture : public SimulatedTestFixture
     void SetUp() override;
 
     /**
-     * Sets the tactic to test
+     * Sets the friendly and enemy tactic to the given tactics, otherwise default to the
+     * halt tactic
+     *
+     * @param friendly_tactic The friendly tactic
+     * @param enemy_tactic The enemy tactic
+     *
+     * @throw invalid_argument if any tactic is invalid
+     */
+    void setTactic(
+        std::shared_ptr<Tactic> friendly_tactic = std::make_shared<StopTactic>(true),
+        std::shared_ptr<Tactic> enemy_tactic    = std::make_shared<StopTactic>(true));
+
+    /**
+     * Sets the friendly tactic to test
      *
      * @param tactic The tactic under test
      *
      * @throw invalid_argument if does not contain an tactic
      */
-    void setTactic(std::shared_ptr<Tactic> tactic);
+    void setFriendlyTactic(std::shared_ptr<Tactic> tactic);
+
+    /**
+     * Sets the tactic for the enemy
+     *
+     * @param tactic The tactic under test (for the enemy)
+     *
+     * @throw invalid_argument if does not contain a tactic
+     */
+    void setEnemyTactic(std::shared_ptr<Tactic> tactic);
 
     /**
-     * Sets the robot id to run tactic on
+     * Sets the friendly and enemy robot IDs
      *
-     * @param robot_id The robot id to run tactic on
+     * @param friendly_robot_id The friendly robot ID
+     * @param enemy_robot_id The enemy robot ID, defaults to UINT_MAX as a flag to not set
+     * an enemy robot ID
      */
-    void setRobotId(RobotId robot_id);
+    void setRobotId(RobotId friendly_robot_id, RobotId enemy_robot_id = UINT_MAX);
```

Comment:
we can have a second function that just takes a friendly_robot_id and no enemy_robot_id and avoid this special value